app/components/ProductCard.tsx

```diff
@@ -0,0 +1,44 @@
+"use client";
+import React from 'react';
+
+interface Product {
+  id: number;
+  title: string;
+  price: number;
+  image: string;
+}
+
+interface ProductCardProps {
+    product: Product;
+    currency: string; // e.g., 'USD', 'EUR', 'JPY'
+  }
+
+  const ProductCard = ({ product, currency }: ProductCardProps) => {
+
+    const formattedPrice = new Intl.NumberFormat('en-IN', {
+        style: 'currency',
+        currency: currency,
+      }).format(product.price);
+
+    const addToCart = () => {
+    console.log(`${product.title} added to cart`);
```

Comment:
Do remove the console log. I know this was just to check, but we should ensure whatever gets merged is production ready kind